justfile

```diff
@@ -24,12 +24,12 @@ enclave := "devnet"
 
 account := "TEST"
 
-name := "Writer"
+name := "Precompiler"
 script-file := name + ".s.sol"
 
 # Space-separated list of script arguments
-script-args := "1000000"
-script-signature := "run(" + replace_regex(script-args, "\\S+\\s?", "uint256,") + ")"
+script-args := "2 3000000 true"
+script-signature := "run(" + replace_regex(script-args, "^(\\S+)\\s+(\\S+)\\s+(\\S+)$", "uint256,uint256,bool") + ")"
```

Comment:
This change should no longer be necessary, now that the `script-signature` automatically conforms to passed in numeric and boolean args